src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -1902,21 +1949,22 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
 #ifdef TARGET_ARM64
         else if ((intrinsic == NI_AdvSimd_LoadAndInsertScalar) || (intrinsic == NI_AdvSimd_Arm64_LoadAndInsertScalar))
         {
-            CorInfoType pSimdBaseJitType = CORINFO_TYPE_UNDEF;
-            var_types   retFieldType     = impNormStructType(sig->retTypeSigClass, &pSimdBaseJitType);
+            var_types pSimdBaseType = TYP_UNDEF;
```

Comment:
nit: This should just be `simdBaseType` since its not a pointer. -- This is a pre-existing issue